### DISCUSSION TITLE
Unable to access boot - Dual RJ45 connection

### DISCUSSION THREAD

**Discussion by sebastiencastermans** [nextcloud/all-in-one#2579] [Questions] [answered]
### Steps to reproduce
1. Reboot nextcloud, the computer or after a power outage

### Expected behavior
Nextcloud should start directly on both networks as do all the other "server apps" I have on this computer. However, I don't need it to start on both, only one of the two (and one in particular) is enough for me.
(Although I would like later to be able to access it via the other network as well (locally) but that's for a little later...)

### Actual behavior
As soon as I restart the computer, with linux on it of course, I can't access Nextcloud. When I go to "the console" from localhost:8079 (port that I have defined), everything is ok and operational. The problem is that I have two RJ45 cables from two different networks that I need on my nextcloud server. Until I unplug one of the two cables, it looks like Nextcloud won't "start" its https server because it doesn't know which network to do it on.

### In detail
I have two separate networks, my home network (192.168.77.xxx) where I need to access the server for other things on it including Octoprint, Openhab, AIO console, etc... (and where I would like to access Nextcloud locally) and the primary network (192.168.37.xxx) from where I access Nextcloud from outside. When I start Nextcloud, in the AIO console, everything is ok, including the apache container. But it is impossible for me to access Nextcloud because "the https server" has not started since there are two networks. I am therefore obliged to temporarily disconnect the cable coming from the home network (77) and then I can reconnect it and there everything is fine. But I wish I could start Nextcloud without having to do this every time!

### Host OS
Ubuntu

#### Nextcloud AIO version
25.0.6

#### Current channel
Stable

**Comment by szaimen** [comment, marked as answer]
Hi, this looks like a general docker problem, not caused by AIO. For help on this, please open a thread in a docker forum.
